lib/northbound.c

```diff
@@ -512,6 +512,7 @@ static int nb_lyd_diff_get_op(const struct lyd_node *dnode)
 	return 'n';
 }
 
+#if 0
```

Comment:
If you are sure this function will not be required anymore, suggest to remove this whole section.

Reply:
They are used in the disabled code below in nb_config_diff. I will add a comment and push an update.